packages/core/src/auth/sso/media/index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <title>AWS Authentication</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link rel="stylesheet" type="text/css" media="screen" href="auth.css" />
+    </head>
+
+    <body>
+        <div class="center-wrapper">
+            <div style="font-size: 3rem" class="center-icon aws-icon">
+                <svg
+                    id="Layer_1"
+                    data-name="Layer 1"
+                    xmlns="http://www.w3.org/2000/svg"
+                    width="6rem"
+                    :height="`${6 * 0.61}rem`"
+                    viewBox="0 0 50 30"
+                >
+                    <path
```

Comment:
This is the AWS icon, attempting to make it an icon and then importing it makes it lose colour :(